examples/connected-car-cluster-sharding/akka-connected-car-streamlet/src/main/scala/connectedcar/streamlets/RawCarDataIngress.scala

```diff
@@ -0,0 +1,75 @@
+package connectedcar.streamlets
+
+import akka.NotUsed
+import akka.stream.scaladsl.Source
+import cloudflow.akkastream.AkkaStreamlet
+import cloudflow.akkastream.scaladsl.RunnableGraphStreamletLogic
+import cloudflow.streamlets.StreamletShape
+import cloudflow.streamlets.avro.AvroOutlet
+import connectedcar.data.ConnectedCarERecord
+
+import scala.util.Random
+import scala.concurrent.duration._
+
+object RawCarDataIngress extends AkkaStreamlet {
```

Comment:
Please rename to RawCarDataGenerator, so it is clear that this just generates some data.

Reply:
done